src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionDocumentEntity.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.time.OffsetDateTime;
+
+@Entity
+@Table(name = "transcription_document")
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
```

Comment:
I don't think we need this EqualsAndHashCode? I was expecting the annotations here to be similar to the other entity added - `AnnotationDocumentEntity`

Reply:
ACCEPT